docs/core/entity/update.md

```diff
@@ -23,6 +23,7 @@ Properties should always only return information from memory and not do I/O (lik
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
+| auto_update | bool | `False` | The device or service that the entity represent have auto update logic, when this is set you can not skip updates.
```

Comment:
The device or service that the entity represents has auto update logic

Reply:
I'd separate the two statements into two sentences to make it more clear.

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/1333